testing/unit/tp/public_key/base.py

```diff
@@ -121,9 +121,9 @@ def generate_ecdsa_payload(key=None, entity_hash=None, entity_hash_signature=Non
     )
 
 
-def generate_header(payload, inputs, outputs):
+def generate_header(payload, inputs, outputs, signer_public_key=None):
```

Comment:
Why not just `signer_public_key = SENDER_PUBLIC_KEY ` ?

Reply:
@DeV1doR, done.